### PR TITLE
Pull request to Dropbears master

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mousetrap-master/tests/test.mousetrap.js [hunk 1]
```diff
@@ -0,0 +1,714 @@
...
+
+        it('binding to alt+shift++ should work as well', function() {
+            var spy = sinon.spy();
+            Mousetrap.bind('alt+shift++', spy);
+
+            KeyEvent.simulate('+'.charCodeAt(0), 43, ['shift', 'alt']);
+            expect(spy.callCount).to.equal(1, 'callback should fire');
+            expect(spy.args[0][1]).to.equal('alt+shift++', 'callback should match "alt++"');
+
+        })
```
[the remark this answers is not in the document]
Comment (James-Wetton) [reply mousetrap-master/tests/test.mousetrap.js]: nice

File: mousetrap-master/mousetrap.min.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+B(c[d+1]).action);q(c[d],m,e,a,d)}}function q(a,b,c,e,d){h._directMap[a+":"+c]=b;a=a.replace(/\s+/g," ");var f=a.split(" ");1<f.length?p(a,f,b,c):(c=B(a,c),h._callbacks[c.key]=h._callbacks[c.key]||[],g(c.key,c.modifiers,{type:c.action},e,a,d),h._callbacks[c.key][e?"unshift":"push"]({callback:b,modifiers:c.modifiers,action:c.action,seq:e,level:d,combo:a}))}var h=this;a=a||v;if(!(h instanceof c))return new c(a);h.target=a;h._callbacks={};h._directMap={};var n={},r,z=!1,t=!1,y=!1;h._handleKey=function(a,
+c,d){var e=g(a,c,d),k;c={};var h=0,l=!1;for(k=0;k<e.length;++k)e[k].seq&&(h=Math.max(h,e[k].level));for(k=0;k<e.length;++k)e[k].seq?e[k].level==h&&(l=!0,c[e[k].seq]=1,f(e[k].callback,d,e[k].combo,e[k].seq)):l||f(e[k].callback,d,e[k].combo);e="keypress"==d.type&&t;d.type!=y||x(a)||e||b(c);t=l&&"keydown"==d.type};h._bindMultiple=function(a,b,c){for(var d=0;d<a.length;++d)q(a[d],b,c)};w(a,"keypress",d);w(a,"keydown",d);w(a,"keyup",d)}if(r){var p={8:"backspace",9:"tab",13:"enter",16:"shift",17:"ctrl",
```
[the remark this answers is not in the document]
Comment (auscompgeek) [reply mousetrap-master/mousetrap.min.js]: Uh, weird. Why is hound-ci poking at this when it should be ignored.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Winston\\AppData\\Local\\Programs\\Python\\Python36\\python.exe"
+}
```
Comment (auscompgeek) [inline .vscode/settings.json]: …

**Comment by auscompgeek** [comment]
Please fix your filenames and submit a new PR. Hopefully Hound CI stops being stupid.

**Pull request closed without merging**